.gitignore

```diff
@@ -17,3 +17,4 @@ PHPCI/Store/MigrationStore.php
 PHPCI/Store/Base/MigrationStoreBase.php
 local_vars.php
 Tests/PHPCI/config.yml
+/nbproject/
```

Comment:
This is not project specific, please add it to your personal "global" ```.gitignore``` file, see [doc](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).